lib/activerecord-import/import.rb

```diff
@@ -412,7 +412,7 @@ def import_helper( *args )
           column_names = self.column_names.dup
         end
 
-        if models.first.id.nil? && column_names.include?(primary_key) && columns_hash[primary_key].type == :uuid
+        if models.first.id.nil? && column_names.include?(primary_key) && [:integer, :uuid].include?(columns_hash[primary_key].type)
```

Comment:
Hey @zdennis 

thanks for the quick reply.

I've noticed a lot of this gem code assume the collection is like the first one. I guess it was the original author (you?) 's decision to make this fair assumption. conditions like this `if args.last.is_a?( Array ) && args.last.first.is_a?(ActiveRecord::Base)` I suppose it would be too much of a check to loop through the array and be counter productive in the end... 

in our case, like probably most users of this gem, we send an array of `Model.new`, so all `id` should be `nil` indeed

about having a mixed collection of ids and nils making me think of another side effect of switching to postgresql. not related to this gem, but just a FYI. setting a ID while inserting a row in postgresql will not update the sequence like mysql does this for us. you need to reset the sequence or risk to have a collision error.